elm/js/javascript.js

```diff
@@ -0,0 +1,28 @@
+var node = document.getElementById('app');
+var storedName = localStorage.getItem('name');
+var socket = io();
```

Comment:
I'm not really sure what you mean? As in copy/paste from the existing client.js or..? Would you like me to change any of it?